python/Ganga/Lib/LCG/Grid.py

```diff
@@ -188,23 +149,14 @@ def __resolve_no_matching_jobs__(cmd_output):
     return glite_ids
 
 
-def list_match(jdlpath, ce=None):
+def list_match(jdlpath, cred_req, ce=None):
```

Comment:
Shouldn't we be documenting the functions which rely on this so that it's easy to see when just looking at one function in isolation?